### DISCUSSION TITLE
spleen_segmentation_3d_lightning tensorboard issues

### DISCUSSION THREAD

**Discussion by seungyunL** [Project-MONAI/tutorials#708] [Q&A]
Hi, I have trouble launching tensorboard in spleen_segmentation_3d_lightning.ipynb.
 - It generates tfevent in log_dir
 - But when I launch tensorboard, it says "No dashboards are active for the current date set"

When I lauhch other example (pytorch-lightning tensorboard examples) it works well.
When I inspect event file running code it shows:

![화면 캡처 2022-05-15 175721](https://user-images.githubusercontent.com/38151724/168465008-1f0088da-0731-4f62-85ed-b5cf0b1b5e11.png)

I have tried both Windows / Ubuntu OS and same problem occurs.
Any solutions?

**Comment by Nic-Ma** [comment]
Hi @yiheng-wang-nv ,

Could you please help verify this issue locally?

Thanks in advance.
